### PR TITLE
feat: APPS-2910 assemble event series page

### PR DESCRIPTION
**Pull request by farosFreed**
Connected to [APPS-2910](https://jira.library.ucla.edu/browse/APPS-2910)

**Page/Pages Created/updated:** [slug].vue 

**Notes:**

Styles:
For the first page I added most styles directly to the page, including some global variables.
Because we are reusing a lot of these styles across pages, I have moved reused styles to global.scss to prevent duplication and keep things DRY

MOBILE:
The mockups for this page show complex behavior for the sidebar content (the previously sticky sidebar needs to split into 2 distinct content blocks and slot into 2 different areas in the column). This work is not 100% done, and will be continued in this ticket https://uclalibrary.atlassian.net/browse/APPS-2946

BLOCKCARDTHREECOLUMN: 
Fixed in this [commit](https://github.com/UCLALibrary/ftva-website/nuxt/pull/39/commits/f3598b35c03a584924e61d4d12f0b064f0ae52d9)
~Currently the links do not work, I believe this is because of the way card-meta implements links using formatLinkTarget, this will likely need to be fixed next week as it cannot be fixed from the page side.~

**Time Report:**

This took me 10-12 hours to build this.

**Checklist:**

-   [X] I added github label for semantic versioning
-   [X] I double checked it looks like the designs
-   [ ] I completed any required mobile breakpoint styling
-   [X] I completed any required hover state styling
-   [X] I included a working spec file
-   [X] I added notes above about how long it took to build this component
-   [ ] UX has reviewed this PR
-   [ ] I assigned this PR to someone to review

[APPS-2910]: https://uclalibrary.atlassian.net/browse/APPS-2910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pages/series/[slug].vue [hunk 1]
```diff
@@ -63,6 +71,86 @@ const parsedCarouselData = computed(() => {
...
+const parsedUpcomingEvents = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!upcomingEvents.value)
+    return []
+
+  // Transform data
+  return upcomingEvents.value.map((item, index) => {
+    return {
+      ...item,
+      image: item.image[0]
```
Comment (pghorpade) [inline pages/series/[slug].vue:85]: Make it fail safe: 
`image: item.image && item.image.length > 0 ? item.image[0] : null`

File: pages/series/[slug].vue [hunk 2]
```diff
@@ -63,6 +71,86 @@ const parsedCarouselData = computed(() => {
...
+const parsedPastEvents = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!pastEvents.value)
+    return []
+
+  // Transform data
+  return pastEvents.value.map((item, index) => {
+    return {
+      ...item,
+      image: item.image[0]
```
Comment (pghorpade) [inline pages/series/[slug].vue:98]: Make it fail safe: 
`image: item.image && item.image.length > 0 ? item.image[0] : null`

File: pages/series/[slug].vue [hunk 3]
```diff
@@ -63,6 +71,86 @@ const parsedCarouselData = computed(() => {
...
+
+// Transform data for Other Series Section
+// This section only shows 3 items max, and prioritizes upcoming events over ongoing
+const parsedOtherSeries = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!otherSeriesUpcoming.value && !otherSeriesOngoing.value)
+    return []
+
+  let otherSeries = otherSeriesUpcoming.value.concat(otherSeriesOngoing.value)
+  // Get first 3 events
```
Comment (pghorpade) [inline pages/series/[slug].vue:111]: we need to further filter the array by removing current series like 
` // Return series without the page's featured event
  otherSeries = otherSeries.filter(({ to }) => !to.includes(route.params.slug) )`

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: pages/series/[slug].vue (hunk 2 above)
Comment (farosFreed) [reply pages/series/[slug].vue:98]: ✅

File: pages/series/[slug].vue (hunk 1 above)
Comment (farosFreed) [reply pages/series/[slug].vue:85]: ✅

File: pages/series/[slug].vue (hunk 3 above)
Comment (farosFreed) [reply pages/series/[slug].vue:111]: ✅

**Review by pghorpade — APPROVED** [review]
👍 well done

**Pull request merged**